static/js/containers/pages/login/LoginForgotPasswordPage.js

```diff
@@ -56,14 +70,62 @@ export class LoginForgotPasswordPage extends React.Component<Props> {
           <div className="row auth-header">
             <h1 className="col-12">Forgot Password</h1>
           </div>
-          <div className="row auth-card card-shadow auth-form">
-            <div className="col-12">
-              <p>Enter your email to receive a password reset link.</p>
+          {this.state.forgotEmailSent ? (
+            <div className="row auth-card card-shadow">
+              <div className="col-12">
+                <h3 className="text-center">Thank You!</h3>
+              </div>
+              <div className="col-12">
+                <p>{this.state.text}</p>
+              </div>
+              <div className="col-12">
+                <p>
+                  <b>
+                    If you do NOT receive your password reset email, here's what
+                    to do:
+                  </b>
+                </p>
+              </div>
+              <div className="col-12">
+                <ol>
+                  <li>
+                    <b>Wait a few moments.</b> It might take several minutes to
+                    receive your password reset email.
+                  </li>
+                  <li>
+                    <b>Check your spam folder.</b> It might be there.
+                  </li>
+                  <li>
+                    <b>Is your email correct?</b> If you made a typo, no
+                    problem, just try to{" "}
```

Comment:
instead of adding space here and at line#107, you can add spaces at place around `reset your password` like {` reset your password `}